js/main.js

```diff
@@ -148,3 +78,65 @@ mobileMenuBtn.addEventListener("click", () => {
 	// toggle mobile menu component
 	mobileMenuComponent.classList.toggle("open-menu");
 });
+
+/* 
+===================================
+Content Management System
+===================================
+*/
+// Helper functions for appending data to DOM elements
```

Comment:
This is meant to keep all the appending as DRY as possible and to run checks. The function works but could probably be improved (moving quickly).